python/sglang/srt/mem_cache/memory_pool.py

```diff
@@ -54,14 +54,15 @@ def __init__(
         device: str,
         enable_memory_saver: bool,
     ):
+
         memory_saver_adapter = TorchMemorySaverAdapter.create(
             enable=enable_memory_saver
         )
 
         self.size = size
         self.max_context_len = max_context_len
         self.device = device
-        with memory_saver_adapter.region():
+        with memory_saver_adapter.region("kv_cache"):
```

Comment:
nit: as mentioned before, what about making it constants instead of repeated strings